mingw-w64-rust/PKGBUILD

```
@@ -237,6 +245,18 @@ package_rust-wasm() {
   install -Dm644 ../LICENSE-{APACHE,MIT} -t "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}-wasm"
 }
 
+package_rust-cygwin() {
+  pkgdesc="MSYS2 target for Rust (mingw-w64) (--target=x86_64-pc-cygwin)"
```

```suggestion
  pkgdesc="MSYS2 target for Rust (mingw-w64)"
```
users should be aware of how to cross-compile with rust :)